<launch>
  <node pkg="teleop_gesture_toolbox" exec="leap_backend_run.sh" output="screen" />

  <node pkg="teleop_gesture_toolbox" exec="static_detection_run.py" output="screen"/>
  <node pkg="teleop_gesture_toolbox" exec="dynamic_detection_run.py" output="screen"/>

  <node pkg="teleop_gesture_toolbox" exec="leap_run.py" output="screen"/>

  <node pkg="coppelia_sim_ros_interface" exec="server_run.py" output="screen"/>

  <!--<node pkg="teleop_gesture_toolbox" exec="main_coppelia_run.py" output="screen"/>-->

</launch>
